qutebrowser/browser/hints.py

```diff
@@ -34,10 +36,15 @@
 from qutebrowser.keyinput import modeman, modeparsers
 from qutebrowser.browser import webelem
 from qutebrowser.commands import userscripts, cmdexc, cmdutils, runners
-from qutebrowser.utils import usertypes, log, qtutils, message, objreg
+from qutebrowser.utils import (usertypes, log, qtutils, message,
+                               objreg)
 from qutebrowser.misc import guiprocess
 
 
+__all__ = ("ElemTuple", "Target", "on_mode_entered", "HintContext",
+           "HintManager")
+
+
```

Comment:
What are you adding this for? `__all__` is only needed with `from ... import *`, which I really never want to use.